.github/workflows/node-zxf-deploy-integration.yaml

```diff
@@ -40,7 +49,7 @@ jobs:
         uses: fjogeleit/http-request-action@bf78da14118941f7e940279dd58f67e863cbeff6 # v1.16.3
```

Comment:
```suggestion
        if: ${{ (inputs.author != "" && inputs.message != "" && inputs.headSha != "") && !cancelled() }}
        uses: fjogeleit/http-request-action@bf78da14118941f7e940279dd58f67e863cbeff6 # v1.16.3
```

Reply:
Had to modify this a little to:
`if: ${{ (inputs.author != '' && inputs.message != '' && inputs.headSha != '') && !cancelled() }}`

Updated.

Reply:
I made this suggestion on. the wrong file though :'(